apps/api/src/app/export/export.service.ts

```diff
@@ -43,6 +45,19 @@ export class ExportService {
         };
       }
     );
+    const allTags = (
+      await this.tagService.getTags({
```

Comment:
You can use `this.tagService.getTagsForUser(userId)` and filter the result by `isUsed`